features/canvas-2d-font-rendering.yml

```
@@ -0,0 +1,19 @@
+name: Canvas 2D advanced font rendering
```

I'm not in love with the name here, but these are a set of newer font properties that group together nicely.

---

Did you look into how developers are referring to this?

FWIW, I think "advanced" font rendering implies a basic font rendering. Is this standing in contrast to something else?

If not and there's no obvious name for this from developers, then I'd suggest something really plain like:

```suggestion
name: Font properties for canvases
```

---

I went with "advanced" because it's not all of the font rendering properties, but only the new ones that have landed in the last few years. I didn't want to call it "new" or "new-ish". 

The other alternative I had was to lump kerning, stretch, and spacing into a feature called "font spacing", then  separate features for direction, caps, and textRendering, but that feels like a bunch of one-off features. 

By grouping them into this one group, we've got a handful of font features that all landed closely enough together that they can be considered a feature.